### ISSUE TITLE
Request: Show Like Percentage next to video thumbnails

### ISSUE THREAD

**Issue by jonas1015119** [Anarios/return-youtube-dislike#463]
A number of extensions used to display the Like/Dislike Percentage next to the videos on the home/channel pages etc, so you could spot bad tutorials or controversial videos before clicking on them. These obviously no longer work, so this extension bringing back that functionality would be much appreciated.

**Comment by HyperSoop** [comment]
great idea, but you may get massively reatelimited by whatever API for making 10 requests per second just scrolling through your reccomends

**Comment by jonas1015119** [comment]
Yeah I was wondering about the number of requests, you would probably need a dedicated caching system

**Comment by cyrildtm** [comment]
Already exists: [Thumbnail Rating Bar for YouTube](https://chrome.google.com/webstore/detail/thumbnail-rating-bar-for/cmlddjbnoehmihdmfhaacemlpgfbpoeb)
Its later versions use cache.

**Comment by Mouadabdelghafouraitali** [comment]
Here's another idea to achieve that without requesting too much from the API, YouTube has added a new feature "**keep hovering to play**", you can place the mouse over the video and the video starts playing, we can do the same thing when the user places the mouse over a video, we can show the likes and dislikes counts

**Comment by TechStudent11** [comment]
> Here's another idea to achieve that without requesting too much from the API, YouTube has added a new feature "**keep hovering to play**", you can place the mouse over the video and the video will starts playing, we can do the same thing when the user places the mouse over a video, we can show the likes and dislikes counts

@Mouadabdelghafouraitali Yes, but that feature is really annoying (at least for me, I've seen criticism about this but not on the scale of the removal of the dislike button). Luckily, YouTube does allow you to disable it (which I did).

So how can someone who's disabled the feature going to get the Like percentage next to the thumbnails.

Another reason is because this only works on the home page. Not searching or anything other than the homepage.
